doc/opencv-logo.md

```diff
@@ -0,0 +1,7 @@
+OpenCV logo has been originally designed and contributed to OpenCV by Adi Shavit in 2006. The graphical part consists of three stylized letters O, C, V, colored in the primary R, G, B color components, used by humans and computers to perceive the world. It is shaped in a way to mimic the famous [Kanizsa's triangle](https://en.wikipedia.org/wiki/Illusory_contours) to emphasize that the prior knowledge and internal processing are at least as important as the actually acquired "raw" data.
```

Comment:
It seems to me it's good to embed an image of the logo in this page

Reply:
Done!

Reply:
I'd like to recommend to move at the top and resize:
```
<img src="opencv-logo2.png" width="256">
```
(or different width)